resources/views/email-verification/done.blade.php

```diff
@@ -0,0 +1,22 @@
+@extends('nodes.api::email-verification.base')
+
+@section('content')
+    <div id="nBox" class="panel panel-default nodes-center">
+        <div class="panel-heading">
+            <h3 class="panel-title">Email verification</h3>
+        </div>
+        <div class="panel-body">
+            @if (Session::has('success'))
+                <div class="alert alert-success text-center" role="alert">
+                    {{ Session::get('success') }}
+                </div>
+            @endif
+            <p><strong>Congratulations!</strong></p>
+            <p>Your email has been now been verified.</p>
```

Comment:
There is some truth in your words, but we don't check other user information in addition to this we can't say that user is verified, only user email is verified that it belongs to that person. But I agree with both options this time, they are both right in some manner. SO if you say to change it - I will